common/app/layout/Sublink.scala

```diff
@@ -268,6 +268,7 @@ case class ContentCard(
     case Some(InlineVideo(_, _, _, Some(_))) => true
     case Some(InlineImage(_)) => true
     case Some(InlineSlideshow(_)) => true
+    case Some(svg@CrosswordSvg(_)) => true
```

Comment:
This is weird. `displayElement` has only four cases so you could nearly get rid of the last match statement, the one with the underscore. But then you aren't covering the case of an `InlineVideo` without a fallback.

I think it's fine enough the way it is though, but you don't need the `svg@`!